src/Symfony/Component/Form/Button.php

```diff
@@ -193,6 +193,22 @@ public function getErrors($deep = false, $flatten = true)
         return new FormErrorIterator($this, array());
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getAllErrors()
+    {
+        return $this->getErrors(true);
```

Comment:
Passing true here is useless.

Reply:
The point of the issue raised is: you will not get all errors without passing `true` which is confusing for the development experience of newer adopters. Passing the parameter is actually the point of the pull request in combination with providing this altogether. After all it is an addition and not a replacement for the current behavior.

Reply:
A button cannot be invalid :).